docs/stencil-docs/localization/localizing-stores.md

```diff
@@ -31,10 +31,27 @@ A BigCommerce storefront can be customized to display in any one language of you
 </div>
 </div>
 
-## Translating theme files via Stencil CLI
+## Localizing theme files
+

```

Comment:
@traci707, can you include an introductory sentence here. We want to avoid empty headers that only contain other headers.